packages/web-components/src/components/content-group/content-group.scss

```diff
@@ -21,13 +21,3 @@
 :host(#{$dds-prefix}-content-group-copy) {
   @include carbon--type-style('body-long-02');
 }
-
-:host(#{$dds-prefix}-content-group-heading) {
-  @include carbon--type-style('expressive-heading-04', true);
```

Comment:
This is repetitive from https://github.com/carbon-design-system/carbon-for-ibm-dotcom/blob/v1/packages/styles/scss/internal/content-group/_content-group.scss#L46-L52